CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 Changelog
 =========
 
-# Unreleased (0.21.0)
+# Unreleased (0.22.0-dev)
+* Nothing yet
```

Comment:
umm do we need this entry? We could just leave it as empty?

Reply:
The previous release has this entry- https://github.com/yarpc/yab/commit/2f7d2420877e28f8dba8854e1cf01044f4c8cdb0
I would like to keep it for the sake of consistency.